scripts/pi-hole/js/ip-address-sorting.js

```diff
@@ -7,7 +7,7 @@
 
 // This code has been taken from
```

Comment:
In which case:

```suggestion
// This code has been adapted from
```

Reply:
I doubt there has been any "adapting" :P

Mostly so that linting passed AFAICT